### PR TITLE
added decrement button

### PR DESCRIPTION
**Pull request by kstolte**

### CODE REVIEW

File: src/app/scoreboard/scoreboard.component.html [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
-  <div class="hero-body">
-    <!-- <div class="container"> -->
-      <div class="tile is-ancestor">
-        <div class="tile is-parent" *ngFor="let color of ['danger', 'info']; let i = index">
-          <app-team-tile
-            [color]="color"
-            (idSet)="idSet($event)"
-            (pointsChange)="pointsChanged($event)"
-            [currentServer]="server">
-          </app-team-tile>
```
Comment (schie) [inline src/app/scoreboard/scoreboard.component.html:11]: why was the formatting changed here?

File: src/app/team-tile/team-tile.component.html [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+    <article class="tile is-child notification is-{{ color }} {{ animatedClass }}">
+        <div class="content">
+            <!-- <p class="title">Team {{ color }}</p> -->
+            <!-- <p class="subtitle">With even more content</p> -->
+            <div class="content">
+                <!-- Content -->
+                <h1 class="score has-text-centered has-text-white-bis is-unselectable">{{ points || 0}}</h1>
+            </div>
+        </div>
+        <div class="content" (click)="decrementBtnClick($event); $event.stopPropagation()">
```
Comment (schie) [inline src/app/team-tile/team-tile.component.html]: I think this should should be a button block instead.
- buttons: http://bulma.io/documentation/modifiers/syntax/
- block jazz: http://bulma.io/documentation/modifiers/responsive-helpers/

File: src/app/team-tile/team-tile.component.ts [hunk 3]
```diff
@@ -34,5 +34,11 @@ export class TeamTileComponent implements OnInit {
     this.points = this.points + 1;
     this.pointsChange.emit({ id: this.id, points: this.points});
   }
-
+  decrementBtnClick(ev: any) {
+    if (this.points === 0) {
+      return;
+    }
+    this.points = this.points - 1;    // have to null out the increment and remove one point from the total.
```
Comment (schie) [inline src/app/team-tile/team-tile.component.ts]: this looks cleaner:
```javascript
this.points = (this.points > 0) ? this.points - 1 : 0;
```

**Review by schie — CHANGES_REQUESTED** [review]
works, but I think it needs some more jazz

File: src/app/scoreboard/scoreboard.component.html (hunk 1 above)
Comment (kstolte) [reply src/app/scoreboard/scoreboard.component.html:11]: no idea I think it was a vscode thing...

File: src/app/team-tile/team-tile.component.ts (hunk 3 above)
Comment (kstolte) [reply src/app/team-tile/team-tile.component.ts]: yea you right....

**Comment by kstolte** [comment]
@schie ready for review again. let me know if you want me to just revert back the old formatting of the scoreboard.component.html

**Review by schie — APPROVED** [review]

**Comment by schie** [comment]
:shipit:

**Pull request merged**